sycl/test/abi/pre-cpp11-abi-test.cpp

```diff
@@ -0,0 +1,50 @@
+// REQUIRES: linux
+// RUN: FileCheck %s --input-file %S/sycl_symbols_linux.dump
```

Comment:
It would be good to have a comment here stating the point of the test and that no new exclusions are allowed to be added.

Reply:
Good point, added comments as you proposed.